src/Support/Composer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Perscom\Support;
+
+class Composer
+{
+    /**
+     * @param string $packageName
+     * @return string|null
+     */
+    public static function getPerscomPackageVersion(string $packageName = 'deschutesdesigngroupllc/perscom-php-sdk'): ?string
```

Comment:
Probably not my place to comment on your PR, but this method gives me the stomach aches.

Use `\Composer\InstalledVersions::getVersion(string $packageName)` instead. It checks the autoloader and `vendor/composer/installed.php` instead. Which is way faster than loading a `json` into memory and decoding it for every request just to add debug info to headers.

Additionally there's a backwards compatibility promise with this method, which is not guaranteed if you load the files yourself from the composer directory.

See: https://getcomposer.org/doc/07-runtime.md#installed-versions

Benchmark using forumify-platform + forumify-perscom-plugin.
This function:
```
Version: "dev-master"
Memory: "1.499696 MB"
Runtime "0.00157 seconds"
```

Using `\Composer\InstalledVersions::getVersion()`
```
Version: "dev-master"
Memory: "0.206544 MB"
Runtime: "0.00080 seconds"
```

About twice as fast and 7x as memory efficient, also scales better in larger projects.

Reply:
This is how I was first attempting to solve the problem. For some reason it wasn't working as intended. I'll re visit to see if I can get it working.